### PR TITLE
Add support for multiple text decorations

### PR DESCRIPTION
**Pull request by aaronjedwards**
## Summary

Adds functionality for specifying multiple values for the `textDecoration` style attribute (Related to #809). 

This brings the `textDecoration` attribute in this library closer in line to  the standard CSS `text-decoration` attribute so it seems like a logical addition (and something that I was looking for in this library).

## Details

In particular, this change allows for specifying either `'underline line-through'` or `'line-through underline'` while disallowing invalid combinations such as `'underline none'`.  

I've updated the Typescript types as well to include `'underline line-through'` or `'line-through underline'` as valid values for the `textDecoration` style attribute.

I am unsure on how to approach implementing a test for this functionality or if one is needed, so any input on that front is welcome.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6574,7 +6569,7 @@ min-indent@^1.0.0:
   resolved "https://registry.yarnpkg.com/min-indent/-/min-indent-1.0.1.tgz#a63f681673b30571fbe8bc25686ae746eefa9869"
   integrity sha512-I9jwMn07Sy/IwOj3zVkVik2JTvgpaykDZEigL6Rx6N9LbMywwUSMtxET+7lVoDLLd3O3IXwJwvuuns8UB/HeAg==
 
-minimatch@^3.0.2, minimatch@^3.0.4:
+minimatch@^3.0.4:
```
Comment (diegomura) [inline yarn.lock:6572]: Im curious why yarn.lock changed here 😅 
Besides that looks great

Comment (aaronjedwards) [reply yarn.lock]: Sorry! I backed out the changes to yarn.lock!

**Comment by aaronjedwards** [comment]
@diegomura I've removed the lock file changes now so if everything else looks good this should be good to go!

File: yarn.lock (hunk 1 above)
Comment (aaronjedwards) [reply yarn.lock:6572]: @diegomura thats weird. I backed out these changes and simply reinstalled the packages and this still comes through. How would you like to resolve this?

**Pull request merged**